mercury/templates/banner.html

```diff
@@ -0,0 +1,35 @@
+{% load static %}
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Mercury Telemetry</title>
+	<!-- CSS -->
+	<link rel="stylesheet" type="text/css" href="{% static 'mercury/style.css' %}">
+</head>
+<body>
+	<nav class="navbar navbar-expand-lg navbar-light bg-white">
+		<div>
+			<ul class="navbar-nav ml-auto">
+				<li class="nav-item">
```

Comment:
Optional: We don't have a logo, but I thought maybe we should put the name of the site in the top bar (NYU Motorsports Telemetry) as the first thing.

Reply:
Do you want to replace Home button with "NYU Motorsports Telemetry".

Reply:
I'm not sure. I was thinking about having the title of the page, but also a link to "Home".

Reply:
Do you want title of page to be a link to Home page

Reply:
Either way is fine with me - I don't know what is best. I think it's fine if its a link to Homepage.

Reply:
Added NYU Motorsports Telemetry as a link to index.html